tests/functional/ext/dict_init_mutate.py

```diff
@@ -0,0 +1,33 @@
+"""Example cases for dict-init-mutate"""
+# pylint: disable=use-dict-literal, invalid-name
+
+base = {}
+
+fruits = {}
+for fruit in ["apple", "orange"]:
+    fruits[fruit] += 1
```

Comment:
Outside the scope of this PR obviously, but shouldn't some kind of warning be raised here? After all, the code doesn't work.

Reply:
oof awful example, I'll change it